app/main/views/frameworks.py

```diff
@@ -741,33 +743,32 @@ def framework_updates_email_clarification_question(framework_slug):
     if framework['clarificationQuestionsOpen']:
         # Send confirmation email to the user who submitted the question
         # No need to fail if this email does not send
-        subject = current_app.config['CLARIFICATION_EMAIL_SUBJECT']
-        tags = ["clarification-question-confirm"]
         audit_type = AuditTypes.send_clarification_question
-        email_body = render_template(
-            "emails/clarification_question_submitted.html",
```

Comment:
Is there a template that should now be deleted?

Reply:
Yes it is - gone.